src/notebooks_service.py

```diff
@@ -196,8 +221,34 @@ def ui(path):
 @authenticated
 def whoami(user):
     """Return information about the authenticated user."""
+    return jsonify(get_user_info(user))
+
+
+@app.route(urljoin(SERVICE_PREFIX, 'servers'))
+@authenticated
+def user_servers(user):
+    """Return a JSON of running servers for the user."""
     info = get_user_info(user)
-    return jsonify(info)
+    servers = info['servers']
+
+    if KUBERNETES:
+        pods = get_pods().items
+        annotations = {
+            pod.metadata.name: pod.metadata.annotations
+            for pod in pods
+        }
+
+        for server_name, properties in servers.items():
+            pod_annotations = annotations.get(
+                properties.get('state', {}).get('pod_name', {})
+            )
+            properties[RENKU_ANNOTATION_PREFIX] = {
+                key[len(RENKU_ANNOTATION_PREFIX+'/'):]: value
```

Comment:
I would set the annotations as `servers[server_name]['annotations'] = {key: value for (key, value) in pod_annotations.items() if key.startswith(RENKU_ANNOTATION_PREFIX)}`

Reply:
👍 for calling it `annotations` - but I would remove the `RENKU_ANNOTATION_PREFIX` from the key anyway, it's not really necessary is it?

Reply:
it might be nice if we decide to allow other prefixes in the future.

Reply:
ok I fixed it up, see the sample JSON below